### ISSUE TITLE
Add Button and Checkbox Components to the Search Bar

### ISSUE THREAD

**Issue by wpslck** [unocorn-squad/job-posting-app#37]
Dependent on tasks #10 and #32.

**Comment by wpslck** [comment]
@jpmaHub the Checkbox component has been completed and merged into the main branch. This task is no longer blocked by #32

**Comment by wpslck** [comment]
I'm closing this task since it's a duplicate of task #26 which was re-opened.
